.travis.yml

```diff
@@ -23,8 +23,8 @@ env:
         - NUMPY_VERSION=stable
         - MAIN_CMD='python setup.py'
         - CONDA_DEPENDENCIES='Cython jinja2'
-        - CONDA_ALL_DEPENDENCIES='Cython jinja2 scipy h5py matplotlib pyyaml scikit-image pandas pytz beautiful-soup ipython mpmath jplephem'
-        - PIP_DEPENDENCIES=''
+        - CONDA_ALL_DEPENDENCIES='Cython jinja2 scipy h5py matplotlib pyyaml scikit-image pandas pytz beautiful-soup ipython mpmath'
+        - PIP_DEPENDENCIES='jplephem'
```

Comment:
@bmorris3 - Don't add it here, as this list is always got installed. Instead add it into the env of each builds where optionals are installed (you can spot them where ``CONDA_DEPENDENCIES=$CONDA_ALL_DEPENDENCIES`` is set).